cartridge/graphql.lua

```diff
@@ -335,6 +339,9 @@ local function execute_graphql(req)
     local resp, err = e_graphql_internal:pcall(_execute_graphql, req)
     if resp == nil then
         log.error('%s', err)
+        if type(err) == 'table' and vars.disable_errstack then
```

Comment:
The same here. `vars.disable_errstack` is always `false`. You didn't add that parameter in init function. I don't understand, why the tests are passed.